packages/toc/style/base.css

```diff
@@ -20,8 +20,22 @@
   height: 100%;
 }
 
+.jp-TableOfContents-placeholder {
+  text-align: center;
+}
+
+.jp-TableOfContents-placeholderContent {
+  padding: 8px;
+}
+
 .jp-TableOfContents .jp-SidePanel-content {
   overflow-y: auto;
+  display: flex;
+  align-items: flex-start;
+}
+
+.jp-TableOfContents .jp-TableOfContents-empty.jp-SidePanel-content {
```

Comment:
Can we simplify this selector? (style calculation performance).

```suggestion
.jp-TableOfContents-empty {
```

Reply:
This is too simplified; with the change applied, the `align-items: center` rule does not take precedence. With `.jp-TableOfContents-empty.jp-SidePanel-content` as the selector, it does.

Reply:
`important!`?

Reply:
I could, but I try to avoid using `!important` as much as possible.